src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportJobRecord.cs

```diff
@@ -26,7 +26,6 @@ public ExportJobRecord(Uri exportRequestUri)
             Status = OperationStatus.Queued;
```

Comment:
I think we need an `IsSystem => false` property

Reply:
Good catch. Added the property.